### PR TITLE
feat: breakout room support

### PR DESCRIPTION
**Pull request by gabiborlea**

### CODE REVIEW

File: src/database/FirehoseConnector.js [hunk 1]
```diff
@@ -228,9 +231,14 @@ class FirehoseConnector {
...
             os,
             browserName,
-            browserVersion
+            browserVersion,
+            isBreakoutRoom,
+            roomId,
+            parentStatsSessionId
         };
 
+        console.log(schemaObj);
```
Comment (andrei-gavrilescu) [inline src/database/FirehoseConnector.js:240]: forgotten debug log?

File: src/app.js [hunk 2]
```diff
@@ -316,7 +316,10 @@ function wsConnectionHandler(client, upgradeReq) {
             ampSessionId: meta.sessionId,
             ampUserId: meta.userId,
             ampDeviceId: meta.deviceId,
-            statsFormat: connectionInfo.statsFormat
+            statsFormat: connectionInfo.statsFormat,
+            isBreakoutRoom: meta.isBreakoutRoom,
+            roomId: meta.roomId,
```
Comment (andrei-gavrilescu) [inline src/app.js:321]: si roomId specific to only breakour rooms? if so maybe we can call it breakoutRoomId so we don't confuse it with the already existing "id" fields, wdyt?

Comment (gabiborlea) [reply src/app.js:321]: Yeah, I'll change it

File: src/database/FirehoseConnector.js (hunk 1 above)
Comment (gabiborlea) [reply src/database/FirehoseConnector.js:240]: yeah, sorry

File: src/app.js [hunk 3]
```diff
@@ -311,12 +311,15 @@ function wsConnectionHandler(client, upgradeReq) {
             endDate: Date.now(),
             endpointId: meta.endpointId,
             startDate: meta.startDate,
-            sessionId: String(meta.meetingUniqueId),
+            sessionId: meta.meetingUniqueId ? String(meta.meetingUniqueId) : null,
```
Comment (gabiborlea) [inline src/app.js:314]: added here this check to avoid having 'undefined' string

**Review by andrei-gavrilescu — APPROVED** [review]

**Pull request merged**